python/test/unit/language/test_core_amd.py

```diff
@@ -1026,6 +1027,58 @@ def kernel(X, Z, BLOCK_M: tl.constexpr, BLOCK_N: tl.constexpr, AXIS: tl.constexp
         else:
             np.testing.assert_equal(z_ref, z_tri)
 
+# ---------------
+# test permute
+# ---------------
+
+
+@pytest.mark.parametrize("dtype_str, shape, perm",
+                         [(dtype, shape, perm)
+                          # TODO: bfloat16
+                          for dtype in ['float16', 'float32']
+                          for shape in [(64, 64), (128, 128)]
+                          for perm in [(1, 0)]])
+def test_permute(dtype_str, shape, perm, device='cuda'):
+    check_type_supported(dtype_str)  # bfloat16 on cc < 80 will not be tested
+    if torch.version.hip is not None:
```

Comment:
Is this skip check just a temporary thing?

IOW, are we sure that this is not fixable?